src/styles/global/print.scss

```diff
@@ -1,8 +1,96 @@
 //
 // CUSTOMIZED PRINT STYLES
+@import 'variables';
 
 @media print {
+  @page {
+    margin-top: 100px;
```

Comment:
Please consider adding a paper size like A4 that the user is able to select portrait and landscape mode.

Reply:
Tried different page size options. It breaks the whole design.

Reply:
True, I tried several approaches with the @page size but they always create unwanted layouts even in Chrome.